lib/src/header_parser/sub_parsers/structdecl_parser.dart

```diff
@@ -33,6 +34,39 @@ class _ParsedStruc {
       (dartHandleMember && config.useDartHandle) ||
       incompleteStructMember;
 
+  // A struct without any attribute is definitely not packed. #pragma pack(...)
+  // also adds an attribute, but it's unexposed and cannot be travesed.
```

Comment:
Please document in the readme which of the two ways clang allows to detect in which ones we don't. Then users have a starting point for manually verifying whether it did the right thing for them.

Reply:
Updated reamde. => https://github.com/mannprerak2/ffigen/tree/packed_struct#user-content-configurations:~:text=Note%3A%20Ffigen%20can%20only%20reliably%20identify,in%20which%20case%20you%20can%20override